model/src/main/proto/feedback_reporting.proto

```diff
@@ -0,0 +1,199 @@
+    // A suggestion for something else about the app.
+    OTHER_SUGGESTION = 3;
+  }
+  SuggestionCategory suggestion_category = 1;
+  // The text input provided by the user.
+  string user_submitted_suggestion = 2;
+}
+
+// A feedback report from the user that brings up an issue found about the app or with a feature in
+// the app that needs to be fixed / improved.
+message Issue {
+  // The category that this issue addresses, as selected by the user.
+  enum IssueCategory {
+    // Unspecified category for an issue report; no reports should have an unspecified category
+    // as users are required to select a category.
+    CATEGORY_UNSPECIFIED = 0;
+    // An issue with lessons or questions.
+    LESSONS_QUESTIONS_ISSUE = 1;
+    // A general issue with the language.
+    LANGUAGE_ISSUE = 2;
+    // An issue with the audio language.
+    LANGUAGE_AUDIO_ISSUE = 3;
+    // An issue with the text language.
+    LANGUAGE_TEXT_ISSUE = 4;
+    // An issue with a topic or the topics list / home screen.
+    TOPICS_ISSUE = 5;
+    // An issue with the user's profile or profile screen.
+    PROFILE_ISSUE = 6;
+    // An issue with something else in the app not listed above.
+    OTHER_ISSUE = 7;
+  }
+  IssueCategory issue_category = 1;
+  // List of checkbox items selected by user in the feedback report.
+  repeated string issues_list = 2;
+  // The user's text input if they select "other" for any option
+  string other_user_input = 3;
+}
+
+// A feedback report from the user after a crash has occurred somewhere in the app.
+message Crash {
+  // The location where the crash occurred.
+  enum CrashLocation {
+    // Unspecified location for a crash report; no reports should have an unspecified location
+    // as users are required to select a response for this.
+    LOCATION_UNSPECIFIED = 0;
+    // Indicates that the crash occurred in the lesson / exploration player.
+    LESSON_PLAYER = 1;
+    // Indicates that the crash occurred in the question player / during a practice session.
+    PRACTICE_QUESTIONS = 2;
+    // Indicates that the crash occurred in the topics list / home screen.
+    TOPICS_LIST = 3;
+    // Indicates that the crash occurred in the options screen.
+    OPTIONS_ACTIVITY = 4;
+    // Indicates that the crash occurred on the user's profile page.
+    PROFILE_ACTIVITY = 5;
+    // Indicates that the crash occurred in a different location not listed above.
+    OTHER = 6;
+  }
+  CrashLocation crash_location = 1;
+  // The user's text input for explaining what happened.
+  string crash_explanation = 2;
+}
+
+// Information collected about the user device's systems.
+message FeedbackReportingSystemContext {
+  // The package name for the specific app version on the device.
+  string package_name = 1;
+  // The package code for the specific app version non the device.
+  string package_version_code = 2;
+  // The ISO-3166 code for the country locale that the feedback report is sent from.
+  string country_locale = 3;
```

Comment:
How will we get this vs. language_locale? Per https://stackoverflow.com/a/3660245 my understanding is that Android only exposes the phone language.

Reply:
Ah darn I didn't realize that. It looks like to get location information will require more device access that I don't think we want to ask for right now (i.e. user has to consent to sharing location data).

I think it's reasonable to omit the country for now, WDYT?

Reply:
Ah darn, I didn't realize that locale was not from location -- I'll keep language_locale and country_locale (these might be different, thought I'm curious how country_locale is determined if not by location), and remove device_language since language_locale is giving the device's language.